app/javascript/vue/repository_item_sidebar/repository_values/ScrollSpy.vue

```diff
@@ -58,7 +60,12 @@ export default {
       }
     });
   },
-
+  computed: {
+    computedNavHeightFromSections() {
+      const navContainerHeight = 33 * this.sections.length;
```

Comment:
why 33?

Reply:
This accounts for the text height of the section + gap between the section and the next one. It allows for dynamic container height with preserving the gaps between sections.